modules/efk-logging-elasticsearch-exposing.adoc

```diff
@@ -40,28 +62,67 @@ metadata:
   name: elasticsearch
   namespace: openshift-logging
 spec:
-  host:
+  host: 
   to:
     kind: Service
     name: elasticsearch
   tls:
-    termination: reencrypt <1>
-    destinationCACertificate: |-   <2>
-      -----BEGIN CERTIFICATE-----
-      [...]
-      -----END CERTIFICATE-----
+    termination: reencrypt
+    destinationCACertificate: <1>
 ----
-<1> Set the termination field to reencrypt.
-<2> Add the contents of the *_admin-ca_* file.
+<1> Add the Elasticsearch CA ceritifcate or use the command in the next step. You do not need to set the `spec.tls.key`, `spec.tls.certificate` and `spec.tls.caCertificate` parameters
+required by some reencrypt routes.
 
-. Use the following command to create the service:
+.. Run the following commnd to add the Elasticsearch CA certificate to the route YAML you created:
 +
 ----
-$ oc create -f my_es_route.yaml
+$cat ./admin-ca | sed -e "s/^/      /" >> <file-name>.yaml
+----  
+
+.. Run the following command to create the route:
++
+----
+$oc create -f <file-name>.yaml
+
+route.route.openshift.io/elasticsearch created
 ----
++
+For an example reencrypt route object, see Re-encryption Termination.
++
+//This line ^^ will be linked when the topic is available.
+
+. Check that the Elasticsearch service is exposed:
 
-. Check that the ElasticSearch service is exposed:
+.. Get the token of this ServiceAccount to be used in the request:
 +
 ----
-curl --silent --insecure -H "Authorization: Bearer $( oc whoami -t )" "https://$( oc get route elasticsearch -o jsonpath='{.spec.host}' ):443/.operations.*/_search" | jq
+$ token=$(oc whoami -t)
 ----
+
+.. Run a command similar to the following to access Elasticsearch through the exposed route:
++
+----
+curl -tlsv1.2 -v --insecure -H "Authorization: Bearer me3IL_WmD_I_McBUm2uhxIayUKped-H3L1njlRRxPlE" "https://elasticsearch-openshift-logging.apps.ntest10.devcluster.openshift.com/.operations.*/_search?size=1" | jq
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0*   Trying 35.170.140.32...
+* TCP_NODELAY set
+  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0* Connected to elasticsearch-openshift-logging.apps.ntest10.devcluster.openshift.com (35.170.140.32) port 443 (#0)
+* ALPN, offering h2
+* ALPN, offering http/1.1
+* ignoring certificate verify locations due to disabled peer verification
+} [5 bytes data]
+* TLSv1.2 (OUT), TLS handshake, Client hello (1):
+} [512 bytes data]
+* TLSv1.2 (IN), TLS handshake, Server hello (2):
+{ [93 bytes data]
+* TLSv1.2 (IN), TLS handshake, Certificate (11):
+{ [1629 bytes data]
+* TLSv1.2 (IN), TLS handshake, Server key exchange (12):
+{ [333 bytes data]
+* TLSv1.2 (IN), TLS handshake, Server finished (14):
+{ [4 bytes data]
+* TLSv1.2 (OUT), TLS handshake, Client key exchange (16):
+} [70 bytes data]
```

Comment:
Please replace the outputs (currently, line 106 through 126) with the following.
```
{
  "took": 49,
  "timed_out": false,
  "_shards": {
    "total": 1,
    "successful": 1,
    "skipped": 0,
    "failed": 0
  },
  <<snip>>
```